packages/RatPack.package/RPTests.class/instance/testCaptureGroups.st

```diff
@@ -0,0 +1,6 @@
+test - routing
+testCaptureGroups
+	
+	RPTestHelper get: '/captureGroups/number/1/word/Test' on: app.
```

Comment:
1 is a somewhat convenient number (or at least somehow the the canonic number that would come out if you asked a lazy developer: Say a number). I would suggest using something else than 0 or 1 as I think it has a higher probability to fail.

Reply:
Fair enough changed it to 123456789